configure.ac

```diff
@@ -251,29 +251,38 @@ AC_TYPE_INT16_T
 AC_TYPE_INT32_T
 AC_TYPE_INT64_T
 
+PKG_CHECK_MODULES([libqb], [libqb],
+		  [if test "x$libqb_PREFIX" = "x"; then libqb_PREFIX=`pkg-config --variable=prefix libqb`; fi])
```

Comment:
I think using `PKG_CHECK_VAR` is best practice (for choosing the right `pkg-config`, but see also below), though that's available since pkg-config 0.28 only. If you want to support earlier versions, this macro can be provided via conditional definition from a private copy of the (very short) implementation in `m4/pkg-compat.m4` or a similar file.
However, I'm not sure about the added value of `libqb_PREFIX`, because `pkg-config` and `AC_CHECK_PROGS` both allow overrides by usual environmental variables anyway.

Reply:
Hey @wferi 

take a look at the new patches, we still need libqb_PREFIX around to detect doxygen2man from specific builds (out of tree), but now it should all be contained with PKG_ and AC_PATH_PROG (yes we need to absolute path where the tool was found, with priority from libqb_PREFIX).

Cheers
Fabio